### PR TITLE
About

### PR DESCRIPTION
**Pull request by DaranDachte**

### CODE REVIEW

File: src/pages/About/About.jsx [hunk 1]
```diff
@@ -1,5 +1,87 @@
...
+            In this project, I decided to study how the <b>Mantine UI</b>{" "}
+            framework works and create an I-Commerce website using it. I liked
+            the Mantine and I hope to do another project with it to consolidate
+            the knowledge gained.{" "}
+            <Anchor href="https://mantine.dev/" target="_blank">
+              Mantine
+            </Anchor>
+          </Text>
+          <Text>
+            I also used the site{" "}
```
Comment (likx2) [inline src/pages/About/About.jsx:44]: I would say website

File: src/pages/About/About.jsx [hunk 2]
```diff
@@ -1,5 +1,87 @@
...
+        <List size="lg" pl="5rem" fw={500}>
+          In my work I use:
+          <List.Item>HTML</List.Item>
+          <List.Item>CSS</List.Item>
+          <List.Item>Java Script</List.Item>
+          <List.Item>React</List.Item>
+          <List.Item>TypeScript...</List.Item>
+          <Text>
+            In this project, I decided to study how the <b>Mantine UI</b>{" "}
+            framework works and create an I-Commerce website using it. I liked
```
Comment (likx2) [inline src/pages/About/About.jsx:36]: E-Commerce

**Review by likx2 — COMMENTED** [review]
Looks great on desktop, but mobile view should be improved

File: src/pages/About/About.jsx (hunk 2 above)
Comment (DaranDachte) [reply src/pages/About/About.jsx:36]: ))) Thank you. Its boring to translate it by hand, so i did it with Google translate, but Google translates E as I.  Great.

File: src/pages/About/About.jsx (hunk 1 above)
Comment (DaranDachte) [reply src/pages/About/About.jsx:44]: TYWM

**Review by likx2 — APPROVED** [review]
Looks good now!

**Pull request merged**